Actions/AL-Go-Helper.ps1

```diff
@@ -2215,7 +2223,11 @@ Function AnalyzeProjectDependencies {
         $no++
     }
 
-    return @($projectsOrder)
+    return [PSCustomObject]@{
+        FullProjectsOrder = $projectsOrder
+        BuildAlso = $buildAlso
```

Comment:
Minor (ignore if you'd like): While you are at it, feel free to rename `BuildAlso` to something like `AdditionalProjects` or similar.

All I know is that `AnalyzeProjectDependencies` doesn't build anything, so I'm actually not sure what `$buildAlso` is 😄

Reply:
I can definitely do that